MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m

```diff
@@ -6,26 +6,38 @@
 //
 
 #import "MatchmakingViewController.h"
+#import "MatchCell.h"
 
-@interface MatchmakingViewController ()
+@interface MatchmakingViewController ()<UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UITableView *MatchTableView;
+@property (nonatomic, strong) UIRefreshControl *refreshControl;
 
 @end
-
+    
 @implementation MatchmakingViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    self.MatchTableView.dataSource = self;
+    self.MatchTableView.delegate = self;
+    self.refreshControl = [[UIRefreshControl alloc] init];
+    [self.MatchTableView insertSubview:self.refreshControl atIndex:0];
+    [self.MatchTableView addSubview:self.refreshControl];
+    self.MatchTableView.rowHeight = 540;
+    MatchCell *cell = [_MatchTableView dequeueReusableCellWithIdentifier:@"MatchCell"];
+    [self.view bringSubviewToFront:cell.matchName];
```

Comment:
Interesting; this seems odd to me. You're creating a single cell and moving a subview to the front. But we'll be creating 4 other cells in cellForRowAtIndexPath; do they need the same treatment?